isis/src/base/apps/csminit/csminit.cpp

```diff
@@ -417,6 +419,43 @@ namespace Isis {
     blobLabel += PvlKeyword("PluginName", pluginName);
     cube->write(csmStateBlob);
 
+    // Try to attach rotations
+    // If a camera fails to generate the original BodyRotation
+    // table will overwrite this one (if there was an original
+    // BodyRotation table)
+    if (ui.WasEntered("ISD")) {
+      QString isdFilePath = ui.GetFileName("ISD");
+      std::ifstream isdFileStream(isdFilePath.toStdString());
+      try {
+        json isd = json::parse(isdFileStream);
+
+        if (isd.contains("body_rotation")) {
+          int bodyFrameCode = isd["body_rotation"]["time_dependent_frames"][0];
+          if (isd["body_rotation"].contains("constant_frames")) {
+            bodyFrameCode = isd["body_rotation"]["constant_frames"][0];
+          }
+
+          SpiceRotation bodyRotation(bodyFrameCode);
+          bodyRotation.LoadCache(isd["body_rotation"]);
+
+          Table bodyTable = bodyRotation.Cache("BodyRotation");
+          bodyTable.Label() += PvlKeyword("Description", "Created by csminit");
+          bodyTable.Label() += PvlKeyword("SolarLongitude",
+            "10"/**toString(spice.solarLongitude().degrees())**/ );
```

Comment:
weird commented out line

Reply:
This was leftover from the original effort to add the body table. Since we don't have a sun position in CSM we can't get the solar longitude unless we extract the sun position table onto the cube like we did with the body rotation. It may make more sense to leave the longitude at 0 instead of 10

Reply:
From some final rounds of testing, we don't need to rotate the look vector into the body fixed reference frame so this was remove